definitions/npm/algoliasearch_v3.x.x/test_algoliasearch-v3.x.x.js

```diff
@@ -1,8 +1,25 @@
 // @flow
-import algoliasearch from 'algoliasearch';
+import algoliasearchImport from 'algoliasearch';
 
-const searchIndex = algoliasearch('', '').initIndex('');
-searchIndex.search('test').then((result) => console.warn(result.hits));
+function syncImportTests() {
+  const searchIndex = algoliasearchImport('', '').initIndex('');
+  searchIndex.search('test').then(result => console.warn(result.hits));
+  // $ExpectError
+  searchIndex.search('test').nonPromiseProperty;
+}
 
-// $ExpectError
-searchIndex.search('test').nonPromiseProperty;
+function requireTests() {
```

Comment:
Is this function testing something different from the `syncImportTests` one?

Reply:
Nope, just the way the library is imported --  `import ...` vs `require(...)`.